src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let sum = days * 40;
+
+  if (sum < 120) {
+    return sum;
+  }
+
+  if (sum >= 280) {
+    sum -= 50;
+
+    return sum;
+  }
+
+  if (sum >= 120) {
+    sum -= 20;
+
+    return sum;
+  }
```

Comment:
The function could be simplified by using `if` with `return` without `else`. After a return statement, the function exits, so an `else` is not needed.